### PR TITLE
Switch LLDP 1: Isl rules

### PR DESCRIPTION
**Pull request by niksv**
Part of #2917

### CODE REVIEW

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java [hunk 1]
```diff
@@ -57,43 +61,66 @@ public Command call() {
...
-                logger.info("Got invalid lldp packet: {} on switch {}. Cookie {}",
-                        input.getPacketInPayload(), switchId, cookie);
-                return;
-            }
-        } catch (Exception exception) {
-            logger.info("Could not deserialize LLDP packet {} on switch {}. Cookie {}. Deserialization failure: {}, "
-                    + "exception: {}", input.getPacketInPayload(), switchId, cookie, exception.getMessage(), exception);
+        if (Cookie.isFlowLldp(cookie)) {
+            handleFlowLldp(input, switchId, cookie);
+        } else if (cookie == LLDP_TRANSIT_COOKIE || cookie == LLDP_INPUT_PRE_DROP_COOKIE) {
```
Comment (sergii-iakovenko) [inline services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java]: Is it the same as isLldpRelated checks?

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java [hunk 2]
```diff
@@ -365,9 +381,11 @@ long installOneSwitchFlow(final DatapathId dpid, final String flowId, final Long
      *
      * @param dpid switch id.
      * @param multiTable flag
+     * @param switchLldp flag
```
Comment (sergii-iakovenko) [inline services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java]: Please, add more information about the flag. "switchLldp" is controversial and unclear by the purpose.

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java (hunk 1 above)
Comment (niksv) [reply services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java]: no. there is no `Cookie.isFlowLldp(value)`

isLldpRelated is "sum" of cases form `if` and from `else`

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java [hunk 3]
```diff
@@ -57,43 +61,66 @@ public Command call() {
         };
     }
 
-    private LldpPacket deserialize(Ethernet eth) {
-        if (eth.getPayload() instanceof LLDP) {
-            return new LldpPacket(eth.getPayload().serialize());
+    private LldpPacket deserializeLldp(Ethernet eth, SwitchId switchId, long cookie) {
+        try {
+            if (eth.getPayload() instanceof LLDP) {
+                return new LldpPacket(eth.getPayload().serialize());
```
[the remark this answers is not in the document]
Comment (niksv) [reply services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/connected/ConnectedDevicesService.java]: fixed

File: services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java (hunk 2 above)
Comment (niksv) [reply services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java]: fixed

**Pull request merged**